staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -2098,3 +2101,27 @@ type NetworkConfiguration struct {
 // +k8s:openapi-gen=true
 type GuestAgentPing struct {
 }
+
+// FlavorMatcher references a flavor that is used to fill fields in the VMI template.
+// +k8s:openapi-gen=true
+type FlavorMatcher struct {
+	// Name is the name of the VirtualMachineFlavor or VirtualMachineClusterFlavor
+	Name string `json:"name"`
+
+	// Kind specifies which flavor resource is referenced.
+	// Allowed values are: "VirtualMachineFlavor" and "VirtualMachineClusterFlavor".
+	// If not specified, "VirtualMachineClusterFlavor" is used by default.
+	//
+	// +optional
+	Kind string `json:"kind,omitempty"`
+
+	// Profile is the name of a custom profile in the flavor. If left empty, the default profile is used.
+	// +optional
+	Profile string `json:"profile,omitempty"`
+
+	// GloballyScoped determines if the Name refers
+	// to a VirtualMachineClusterFlavor or VirtualMachineFlavor.
+	// Defaults to true.
+	// +optional
+	GloballyScoped *bool `json:"globallyScoped,omitempty"`
```

Comment:
we don't need this anymore now that the `Kind` has been added

Reply:
True, I missed this.
Removed.